### PR TITLE
Page imply + test

### PR DESCRIPTION
**Pull request by maxence-charriere**
Implementation of the page function.
Jira: https://segment.atlassian.net/browse/EPD-2745
API: https://learn.customer.io/api/#apipageview_event

@tsholmes
@segment-integrations/integrations

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -46,6 +46,25 @@ Customerio.prototype.loaded = function() {
...
+ * 
+ * @api public
+ * @param {Page} page
+ */
+Customerio.prototype.page = function(page) {
+  var name = page.name();
+  var data = page.properties();
+
+  if (!name) {
+    name = data.url;
```
Comment (Peripheral1994) [inline lib/index.js]: Does this handle if data.url is undefined? The API seems to require it - perhaps an early return? Also, you can use <Facade>.proxy() to pull nested values. :)

Example here: 
var name = page.name() || page.proxy('properties.data.url');
if (!name) return;

Comment (Peripheral1994) [inline lib/index.js]: Also worth noting: .proxy() is a safe way to get around an Uncaught if, for example, properties.data didn't exist. :)

Comment (maxence-charriere) [inline lib/index.js]: url should be defaulted if not set.

**Comment by maxence-charriere** [comment]
@tsholmes @segment-integrations/integrations 
Hey guys,
I set up a environment to test the modifications.
Look like data go to customer.io
screen shot 2017-05-08 at 4 28 54 pm

Let me know if some other things should be modified or tested.

File: test/index.test.js [hunk 2]
```diff
@@ -65,6 +65,22 @@ describe('Customer.io', function() {
...
+        analytics.page();
+        analytics.called(window._cio.page);
+      });
+
+      it('should call page with name', function() {
+        analytics.page('home');
+        analytics.called(window._cio.page, 'home');
+      });
+    });
+
```
Comment (tsholmes) [inline test/index.test.js]: Could we add a test here that makes sure the page properties get sent through?
These are sent like `analytics.page('home', { visitNumber: 3 })`

**Comment by hankim813** [comment]
@maxence-charriere lgtm -- i'd just add a release commit on top of this before squashing and merging

bump `package.json`'s minor version
update `HISTORY.md`
`git commit -m 'Release x.x.x'

**Review by hankim813 — APPROVED** [review]

File: lib/index.js [hunk 3]
```diff
@@ -46,6 +46,20 @@ Customerio.prototype.loaded = function() {
...
+ * Page.
+ *
+ * https://learn.customer.io/api/#apipageview_event
+ *
+ * @api public
+ * @param {Page} page
+ */
+
+Customerio.prototype.page = function(page) {
+  var name = page.name() || page.url();
```
Comment (hankim813) [inline lib/index.js]: @maxence-charriere actually i think it's probably better to use `page.path()`? what is the recommend implementation if you were to use this `_cio.page()` method natively without segment?

Comment (hankim813) [inline lib/index.js]: I only ask for UI aesthetics here... since may `urls` are super long.

Comment (maxence-charriere) [inline lib/index.js]: they are using a full url as name in all the examples from their api doc.

Comment (hankim813) [inline lib/index.js]: okay sounds good!

**Pull request merged**